spec/templatingBehaviors.js

```diff
@@ -518,6 +518,36 @@ describe('Templating', function() {
         expect(testNode.childNodes[0]).toContainHtml("begin<span>the name is beta</span>end");
     });
 
+    it('Should accept a "nodes" option that gives the template nodes, and able to use the same nodes for multiple bindings', function () {
+      testNode.innerHTML = "<div data-bind='template: { nodes: testNodes, data: testData1, bypassDomNodeWrap: true }'></div><div data-bind='template: { nodes: testNodes, data: testData2, bypassDomNodeWrap: true }'></div>";
```

Comment:
Yes I've actually copied part of the code. I've checked that setting too and haven't found any other usages so I guess it was used in some previous versions and it's left there accidentally. I didn't remove it to keep the focus of this PR.